pkg/operator/secrets/volume_mount.go

```diff
@@ -41,6 +41,9 @@ type VolumeMount struct {
 	// DirName is the name of the directory under VolumeMountDir in which to
 	// mount this SecretSource.
 	DirName string
+	// AbsolutePath stores the absolute path to use instead of the generated path
+	// with /vt/secret as the prefix
+	AbsolutePath string
```

Comment:
This had to be added to allow mounting files to a specific path instead of the `/vt/secrets` where they usually are. For the Vtadmin web config, the path has to be `/vt/web/vtadmin/build/config`